sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/src/LruCache.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Data.SchemaRegistry.ApacheAvro
+{
+    internal class LruCache<K,V>
```

Comment:
You may want to consider adding a small blurb about the approach that you're taking.   It took me a couple of passes to connect the implementation with the concept .